sycl/include/sycl/detail/reduction_forward.hpp

```diff
@@ -46,11 +46,6 @@ enum class strategy : int {
   multi,
 };
 
-// Reductions implementation need access to private members of handler. Those
-// are limited to those below.
-inline void finalizeHandler(handler &CGH);
-template <class FunctorTy> void withAuxHandler(handler &CGH, FunctorTy Func);
```

Comment:
`withAuxHandler` declaration should have been removed earlier when `postProcess` was introduced instead of it.